lib/cdk_erigon.star

```diff
@@ -4,6 +4,7 @@ CDK_ERIGON_TYPE = struct(
     sequencer="sequencer",
     rpc="rpc",
 )
+CDK_ERIGON_CMD = "cdk-erigon --config /etc/cdk-erigon/config.yaml"
```

Comment:
What's the benefit of defining the cmd in a variable? I'm maybe overlooking something 👀

Reply:
The final approach for Erigon L1 recovery is to set it in the config file params, but I started trying to achieve it command line, during these tests I moved the cmd to a var, because I had some logic there.
When rolling back, it just remained that way. I think it does not hurts, and to me it always looks a bit better to have the all that kind of super specific things as much isolated as possible, but can be changed as it was as well, no problem at all!

Reply:
Ok sounds good to me :)